tests/integration/ceph_base_object_test.go

```diff
@@ -199,6 +201,13 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 				time.Sleep(5 * time.Second)
 				continue
 			}
+			logger.Info("objectstore status is", objectStore.Status)
+			if objectStore.Status.BucketStatus.Health == cephv1.ConditionFailure && CephObjectStoreCIRetryCount == 0 {
```

Comment:
To retry, what if we increase the loop count to 10 instead of 4 up on line 196? Then there is no need to keep track of this retry count.
```suggestion
			if objectStore.Status.BucketStatus.Health == cephv1.ConditionFailure {
```

Reply:
ok